internal/controller/nonadminrestore_controller_test.go

```diff
@@ -357,6 +364,9 @@ var _ = ginkgo.Describe("Test full reconcile loop of NonAdminRestore Controller"
 						LastTransitionTime: metav1.NewTime(time.Now()),
 					},
 				},
+				QueueInfo: &nacv1alpha1.QueueInfo{
```

Comment:
I do not know what happens if we tell Velero to restore a backup in progress. I suspect it fails 

To make this test scenario more real, would change `VeleroBackup.Status` to phase completed, have CompletionTimestamp and `QueueInfo.EstimatedQueuePosition` to be zero